python-package/lightgbm/engine.py

```diff
@@ -408,6 +415,8 @@ def cv(params, train_set, num_boost_round=100,
     callbacks : list of callables or None, optional (default=None)
         List of callback functions that are applied at each iteration.
         See Callbacks in Python API for more information.
+    disp_train_loss : bool, (default=False)
```

Comment:
I think that `show_train_loss` will be better. There are already `show_stdv` param, for instance.
`optional` word is missed after the comma.

Reply:
Thank you for your advice. Revised the argument name and the comment of show_train_loss.